lib/assets/javascripts/cartodb/public_table/table_public.js

```diff
@@ -69,16 +69,6 @@
         this.authenticated_user.fetch();
       }
 
-      // Navigation
-      this.header = new cdb.open.PublicHeader({
```

Comment:
Not that I've noticed, nothing that stands out as detrimental at least. I suspect the construction happens too quickly once it's kicked off for its ordering to affect the UI for good or bad.